Parse-Dashboard/parse-dashboard-config.sample.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Se don't need this file, right?

Reply:
Yeah, sorry about that... It was just to me not to lose the original reference